client/my-sites/stats/post-trends/index.jsx

```diff
@@ -127,22 +115,18 @@ class PostTrends extends React.Component {
 	};
 
 	getMonthComponents = () => {
-		var i,
-			months = [],
-			startDate;
+		const { streakData } = this.props;
+		// Compute maximum per-day post count from the streakData. It's used to scale the chart.
+		const maxPosts = reduce( streakData, ( max, count ) => ( count > max ? count : max ), 0 );
```

Comment:
FFTI: could use _.[max](https://lodash.com/docs/4.17.4#max)

Reply:
Good point, converted to `max( values( streakData ) )`, as `streakData` is an object with shape `{ 'date1': count1, 'date2': count2, ... }`.